helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml

```diff
@@ -33,6 +33,18 @@ spec:
         severity: page
         team: cabbage
         topic: cilium
+    - alert: CiliumAPITooSlow
+      annotations:
+        description: '{{`Cilium API processing time is >50s pod="{{ $labels.pod }}" node="{{ $labels.node }}" method="{{ $labels.method}}" path="{{ $labels.path }}"`}}'
+        opsrecipe: cilium-performance-issues/
+      expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)
+      for: 20m
+      labels:
+        area: managedservices
```

Comment:
```suggestion
        area: platform
```

or is it kaas?

Reply:
It's platform, I just opy/pasted from the one below